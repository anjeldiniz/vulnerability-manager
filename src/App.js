import React from 'react';
import AuthProvider from './contexts/authProvider';
import GlobalStyle from './styles/global';
import RoutesApp from './routes/index';

function App() {
  return (
    <AuthProvider>
      <RoutesApp />
      <GlobalStyle />
    </AuthProvider>
  );
};

export default App;
