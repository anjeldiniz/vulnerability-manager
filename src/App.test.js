import { render } from '@testing-library/react';
import App from "./App";

describe("Test the App", () => {
  it("Should match the snapshot", () => {
    const { asFragment } = render(<App />)
    expect(asFragment()).toMatchSnapshot();
  })
})
