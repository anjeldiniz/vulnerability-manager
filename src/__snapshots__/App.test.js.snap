// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the App Should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-dkrFOg ifroPs"
  >
    <h1
      class="sc-eDvSVe dYFFqZ"
    >
      Vulnerability Controller
    </h1>
    <div
      class="sc-hLBbgP dEurWW"
    >
      <input
        class="sc-bcXHqe gqVlpo"
        data-testid="email"
        placeholder="E-mail"
        type="email"
        value=""
      />
      <input
        class="sc-bcXHqe gqVlpo"
        data-testid="password"
        placeholder="Password"
        type="password"
        value=""
      />
      <label
        class="sc-gKPRtg boeerd"
      />
      <button
        class="sc-gswNZR fKyoNG"
      >
        Enter
      </button>
      <label
        class="sc-jSUZER aOBqO"
      >
        Don't have an account?
        <strong
          class="sc-iBYQkv irgwum"
        >
          <a
            href="/signup"
          >
            Â Register
          </a>
        </strong>
      </label>
    </div>
  </div>
</DocumentFragment>
`;
